diff --git a/node_modules/sequelize/lib/sequelize.js b/node_modules/sequelize/lib/sequelize.js
index b074bee..cf1cd38 100644
--- a/node_modules/sequelize/lib/sequelize.js
+++ b/node_modules/sequelize/lib/sequelize.js
@@ -1,5 +1,4 @@
 'use strict';
-
 const url = require('url');
 const path = require('path');
 const retry = require('retry-as-promised');
@@ -173,16 +172,26 @@ class Sequelize {
     let config;
 
     if (arguments.length === 1 && typeof database === 'object') {
+      console.log("this first option getting hit")
       // new Sequelize({ ... options })
       options = database;
       config = _.pick(options, 'host', 'port', 'database', 'username', 'password');
     } else if (arguments.length === 1 && typeof database === 'string' || arguments.length === 2 && typeof username === 'object') {
       // new Sequelize(URI, { ... options })
+      console.log("you are here")
+      console.log(arguments)
 
       config = {};
       options = username || {};
-
-      const urlParts = url.parse(arguments[0], true);
+      console.log("getting to this point")
+      //console.log(arguments['1'])
+      console.log(Object.keys(arguments))
+      console.log(Object.values(arguments))
+
+      const argumentsChanged = Object.values(arguments[1])
+      const urlParts = url.parse(argumentsChanged[0], true);
+      console.log("now to here")
+      console.log(urlParts)
 
       options.dialect = urlParts.protocol.replace(/:$/, '');
       options.host = urlParts.hostname;
